lib/src/disk_usage/disk_usage_view.dart

```diff
@@ -48,23 +51,61 @@ class _DiskUsageViewState extends State<DiskUsageView> {
         label: const Text("Export Logs"),
         icon: const Icon(Icons.bug_report),
         onPressed: () async {
-          // i would use "saveFile" but for some reason it crashes
-          final dir = await FilePicker.platform.getDirectoryPath();
-          if (dir == null) return;
-          final path = p.join(dir, "ditto_log.txt");
-          await DittoLogger.exportLogs(path);
-          _showSnackbar("Logs exported to $path");
+          try {
+            String exportPath;
+            
+            if (Platform.isIOS) {
```

Comment:
We can't import `Platform` in library code, since it will break web builds

Reply:
@cameron1024 damn it I forgot about this when I did the permissions thing.  Let me fix it - I need to write myself a note on this.

Reply:
fixed